spec/controllers/application_controller_spec.rb

```diff
@@ -36,7 +36,7 @@ def show
     describe "when a new version of the application is deployed" do
 
       it "busts the ETAG" do
-        controller.stub(:etag_seed).and_return "CURRENT_SHA1"
+        allow(controller).to receive(:etag_seed).and_return("CURRENT_SHA1")
```

Comment:
I assume you changed this to get rid of the deprecation warning, if so there are other uses of `stub` in this file you could change now as well.